gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -210,12 +203,6 @@ class GateConfig extends RedisHttpSessionConfiguration {
     createClient "clouddriver", ClouddriverService
   }
 
-  @Bean
-  @ConditionalOnProperty("services.keel.enabled")
-  KeelService keelService(OkHttpClientProvider clientProvider) {
```

Comment:
This was a duplicate bean. There's another one defined later in this file without the `OkHttpClientProvider` argument (which is ignored in this method anyways).